Plots/MapProjections/NCL_native_2.py

```diff
@@ -47,49 +45,63 @@
 # Generate axes using Cartopy and draw coastlines
 ax = plt.axes(projection=ccrs.Mercator())
 ax.coastlines(linewidths=0.5)
-ax.add_feature(cfeature.LAND, facecolor='lightgray')
+ax.add_feature(cfeature.LAND, facecolor="lightgray")
 
 # Set extent to include latitudes from 34 to 52 and longitudes from 128
 # to 144
 ax.set_extent([128, 144, 34, 52], ccrs.PlateCarree())
 
 # Contourf-plot data (for filled contours)
-pt = t.plot.contourf(ax=ax, transform=ccrs.PlateCarree(), vmin=0, vmax=70,
-                     levels=15, linewidths=0,cmap='inferno',
-                     cbar_kwargs={
+pt = t.plot.contourf(
+    ax=ax,
+    transform=ccrs.PlateCarree(),
+    vmin=0,
+    vmax=70,
+    levels=15,
+    cmap="inferno",
+    cbar_kwargs={
         "extendrect": True,
         "orientation": "vertical",
-        "ticks": np.arange(0,71,5),
-        "label": '',
-        })
-
-
-
-# Contour-plot data (for borderlines)
-t.plot.contour(ax=ax, transform=ccrs.PlateCarree(),
-                    levels=12, linewidths=0.5, cmap='k')
+        "ticks": np.arange(0, 71, 5),
+        "label": "",
+    },
+)
 
 # Draw gridlines
-gl = ax.gridlines(crs=ccrs.PlateCarree(), draw_labels=True, dms=False, x_inline=False, y_inline=False, 
-                  linewidth=1, color='k', alpha=0.5)
+gl = ax.gridlines(
+    crs=ccrs.PlateCarree(),
+    draw_labels=True,
+    dms=False,
+    x_inline=False,
+    y_inline=False,
+    linewidth=1,
+    color="k",
+    alpha=0.5,
+)
 
 # Manipulate latitude and longitude gridline numbers and spacing
 gl.ylocator = mticker.FixedLocator(np.arange(128, 145, 20))
 gl.xlocator = mticker.FixedLocator(np.arange(34, 53, 20))
-
 gl.top_labels = False
 gl.right_labels = False
 gl.xlines = False
 gl.ylines = False
-gl.xlocator = mticker.FixedLocator([130,134,138,142])
-gl.ylocator = mticker.FixedLocator([36,38,40,42,44,46,48,50])
-gl.xlabel_style = {'rotation':0}
-gl.ylabel_style = {'rotation':0}
-
-# Use geocat.viz.util convenience function to add titles to left and right
-# of the plot axis.
-gvutil.set_titles_and_labels(ax, maintitle="Native Mercator Projection",
-                             lefttitle="Sfree surface deviation", righttitle="m")
+gl.xlocator = mticker.FixedLocator([130, 134, 138, 142])
+gl.ylocator = mticker.FixedLocator([36, 38, 40, 42, 44, 46, 48, 50])
+gl.xlabel_style = {"rotation": 0, "size": 15}
+gl.ylabel_style = {"rotation": 0, "size": 15}
+
+"""
+Currently, Cartopy does not have the ability to add tick marks on projections. 
+The GeoCAT Team is working to update the GeoCAT-viz convenience functions to 
+account for this 
+"""
```

Comment:
On another PR (#143) I just drew the latitude and longitude lines in place of ticks then I left a comment explaining why I did that and how to remove the lines if they aren't wanted. Since this projection is rectangular, its probably less important than for a non-rectangular projection but might be worth considering.